compiler/src/main/scala/org/scalajs/core/compiler/GenJSCode.scala

```diff
@@ -1998,7 +1998,24 @@ abstract class GenJSCode extends plugins.PluginComponent
           isRawJSCtorDefaultParam(sym)
         } else {
           sym.hasFlag(reflect.internal.Flags.DEFAULTPARAM) &&
-          isRawJSType(sym.owner.tpe)
+          isRawJSType(sym.owner.tpe) && {
+            /* If this is a default parameter accessor on a
+             * ScalaJSDefinedJSClass, we need to know if the method for which we
+             * are the default parameter is exposed or not.
+             * We do this by looking up the de-mangled name in the owner. Note
```

Comment:
"de-mangled" makes me think about `unencodedName` much more than what's happening here. Consider something like "We do this by removing the `$default` suffix from the method name, and looking up a member with that name in the owner."